integration/wasm/src/memspike.rs

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! `memspike`'s primary intention is to trigger memory ballooning in
+//! VM-based keeps. This will help test the ballooning itself as well
+//! as memory pinning for SEV.
+
+fn main() {
+    let _alloc: Vec<u8> = Vec::with_capacity(16_000_000);
```

Comment:
When we run these tests are they being compiled with optimizations? Because if so, then I suspect this line is being entirely optimized away and thus can't cause a memory spike. We'd want to use `std::hint::black_box` to force the optimizer not to remove it (although black_box is unstable, I think it would be trivial to stabilize).

Reply:
Yeah, it's not running optimized... `black_box` would be cool to be stabilized.
Will fix the test with `try_reserve()`.